sig-node/sig-node-contributor-ladder.md

```diff
@@ -0,0 +1,116 @@
+
+## SIG Node reviewers and approvers
+
+SIG Node maintains Kubernetes components that are running large amounts of workloads across clouds, and standalone deployments. Changes in these components have a drastic effect on workload availability. This sets a high bar for SIG Node contributions and puts a lot of emphasis on reliability and security of these changes. Practices and policies of earlier stages of Kubernetes need to be adjusted to the current maturity level of Kubernetes.
+
+This document explains SIG Node guidelines and requirements to become a reviewer or approver. These requirements are written in the spirit of established Kubernetes [membership document](../community-membership.md) and attempt to help contributors, who want to get reviewer and approver status, to formalize the implicit requirements that the SIG aspires to use when making decisions.
+
+Ultimately, membership in the Kubernetes community, as well as reviewer and approver status is a way to establish trust. Starting with permission to run tests, all the way to deciding on the SIG and Kubernetes overall roadmap for key APIs. No amount of documentation can formalize how this trust is established. This document outlines ways to demonstrate expertise of the code base, sound judgement on decision tradeoffs, end user advocacy, care for community, and ability to work as a distributed team.
+
+As a high level guidance, SIG Node reviewers and approvers must have a bias to say “no” in reviews and design discussions, and an initial bias towards coding over reviews to demonstrate a baseline for konwledge of code base.  This high level guidance is dictated by the maturity stage of Kubernetes. It is better to have inefficiencies or known bugs than break existing workloads. It is also important to keep the codebase healthy and maintainable.
+
+We anticipate providing opportunities for emerging contributors to make these contributions where they express their desire for long term maintenance.  This is facilitated during release planning when pairing approvers with contributors for new feature work and/or e2e health with more testing focused specific contributions as we maintain CI.  Helping maintain our CI and test suite is a great way of demonstrating competency!
+
+### Reviewer
+Reviewer status is an indication that the person is committed to the SIG activities, technically sound, has accumulated enough context, and is overall trustworthy to help inform approvers and aid contributors. The requirements listed in the [membership document](../community-membership.md#reviewer) highlight this as well.
+
+The following is the breakdown of requirements listed in the membership document broken down to these categories:
+
+- Committed - proof of sustained contributions
+- member for at least 3 months
+- Technically sound
```

Comment:
Maybe use the "Demonstrates technical depth" from my suggestion above? A person can't really be technically sound. :)